### ISSUE TITLE
Node Information not displayed for kubeadm nodes

### ISSUE THREAD

**Issue by toschneck** [kubermatic/dashboard#2771]
**What happened?**:
After a node joined the cluster, the node doesn't show up in the cluster overview.
Informer version the node information have been shown.
```
$ k get nodes -o wide
NAME                          STATUS   ROLES    AGE    VERSION   INTERNAL-IP   EXTERNAL-IP   OS-IMAGE             KERNEL-VERSION       CONTAINER-RUNTIME
tobi-baremetal-sim-worker-1   Ready    <none>   173m   v1.18.8   10.2.10.210   <none>        Ubuntu 18.04.5 LTS   4.15.0-118-generic   docker://19.3.6
```
![image.png](https://images.zenhubusercontent.com/5b6454d031c5fe7463837d8b/d8d56701-0106-45b3-8a0c-b66c9ae033f4)
**What should happen?**:
It should display the node information like e.g. at imported cluster nodes or in previous Kubermatic versions
(besides of the duplicated kubeadm join command)
![image.png](https://images.zenhubusercontent.com/5b6454d031c5fe7463837d8b/3fbe98a0-3df5-4249-a385-a803bcac4ec6)

**How to reproduce?**:
Create kubeadm cluster and add a kubeadm node

**Environment**
- UI Version: ``
- API Version:
- Domain:
  - [ ] run
  - [x] run-2
  - [ ] dev
  - [ ] locally
- Others:

**Comment by toschneck** [comment]
@maciaszczykm awesome speed guys! Thanks a lot :+1:

**Comment by toschneck** [comment]
@maciaszczykm tested it, the nodes are now visible, but could we remove the duplicated kubeadm info as well? Or should we create an extra issue for this?
![image](https://user-images.githubusercontent.com/4659261/98983156-af96aa00-2520-11eb-9cf8-153021e9e1d3.png)

**Comment by toschneck** [comment]
Still open issue
![image](https://user-images.githubusercontent.com/4659261/107019080-8abd7600-67a1-11eb-93d7-cc6590da7c4c.png)

**Comment by maciaszczykm** [comment]
@toschneck On which version?

**Comment by toschneck** [comment]
@maciaszczykm `weekly-2021-03`

**Comment by toschneck** [comment]
@maciaszczykm any news? It's still present at 2.16.2
![image.png](https://images.zenhubusercontent.com/5b6454d031c5fe7463837d8b/5e948cf3-9f2c-4833-bf15-5a85f5db2641)

**Comment by maciaszczykm** [comment]
I'll check.

**Comment by maciaszczykm** [comment]
@toschneck I've prepared a fix for that, it will go into v2.16 too.

**Comment by toschneck** [comment]
awesome! Thanks a lot!
